#ifndef MENU_SEARCH_H
#define MENU_SEARCH_H

#include "navibasewnd.h"
#include "ui_menu_Search010000.h"

namespace Ui {
class SearchMenu;
}

class Menu_Search : public NaviBaseWnd
{
    Q_OBJECT

public:
    Menu_Search(QWidget *parent = NULL);
    ~Menu_Search();

public slots:
    void searcharoundSlot();
    void keywordSlot();
    void backSlot();
    void searchcrossSlot();
    void searchcategorySlot();
private:
    Ui::SearchMenu ui;
};

#endif // MENU_SEARCH_H
